services/emailservice.js

```diff
@@ -0,0 +1,87 @@
+
+/* 
+Rudimentary email service
+Currently creates emails for account creation, password changes, and error message if something else is attempted.
+
+TODO:
+- replace gmail with SMTP server
+- incorporate userID, so a client's username/password/email can be fetched with a GET request
+------ or will this already be done in some other piece of code?
+- make transporter password a function parameter, instead of just having the email password sit there in plain sight
+------ transporter's fields would have to be created in an outside function
+------ then, just do:
+------ let transporter = nodemailer.createTransport(setTransporterFields(password))
+
+NOTES:
+- got rid of "await" that was in front of transporter.sendMail(mail);
+------ while the code successfully sends emails now, info.messageId now becomes undefined. 
+------ does this matter? is there some other reason why it's better to have await?
+*/
+
+const nodemailer = require("nodemailer");
+
+async function main(){
+  // sets up info for email medium being used
+  let transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: '523pediatrics@gmail.com',
```

Comment:
We will add this to the environment (.env file at the base of the project adds to the environment variables), and will be accessed through `process.env.EMAIL_USER` and `process.env.EMAIL_PASSWORD`.
```suggestion
      user: process.env.EMAIL_USER,
      password: process.env.EMAIL_PASSWORD
```